source/services/github.rst

```diff
@@ -0,0 +1,63 @@
+Github
+======
+
+Every project needs a source code repository and we usually use Github to host ours. This assumes
+that you'll be creating the repo in the Caktus organization, but the process is the same for any
+other organization... you'll just need the permissions to do so.
+
+#. First, let's set up a new project locally using the Django Project Template:
+
+   .. code-block:: bash
+
+      $ mkvirtualenv -p `which python3` myproject
+      (myproject)$ pip install Django
+      ... Django installs ...
+      (myproject)$ django-admin.py startproject \
```

Comment:
This duplicates instructions in the project template, and is likely to get out of sync with it. It's also not really relevant to github - maybe we should just skip over this with "once you have your local project set up, ..." and then continue on with "git init"?